devops/deploy.sh

```diff
@@ -42,16 +47,17 @@ LimitNOFILE=4000
 WantedBy=multi-user.target
 EOF
 
-systemctl --user enable arena
+systemctl --user enable $RELEASE
 
 cat <<EOF >$HOME/.env
 PHX_HOST=${PHX_HOST}
```

Comment:
```suggestion
PHX_HOST=${PHX_HOST}
DATABASE_URL=${DATABASE_URL}
```

Reply:
I believe we should add the database URL, because this `.env` file is sourced at runtime. 

Meaning that these environment variables are loaded when the compiled binary is executed by systemd. Not passing this variable would result in an error, if I'm not mistaken.